packages/emf/backend/sirius-components-emf/src/main/java/org/eclipse/sirius/components/emf/MigrationData.java

```diff
@@ -0,0 +1,31 @@
+/*******************************************************************************
+ * Copyright (c) 2024 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.sirius.components.emf;
+
+/**
+ * POJO for MigrationData.
+ *
+ * @author mcharfadi
+ */
+public record MigrationData(String lastMigrationPerformed, String migrationVersion) {
+
+    public static final String JSON_OBJECT_ROOT = "migration";
+
+    @Override
+    public String toString() {
+        return "MigrationData{" +
+                "lastMigrationPerformed='" + lastMigrationPerformed + '\'' +
+                ", migrationVersion='" + migrationVersion + '\'' +
+                '}';
+    }
```

Comment:
You don't have to customize the toString methods of record like we did with our old data structure, the default version is good enough for our use cases.

Reply:
Done